github/handlers/EventHandler.ts

```diff
@@ -118,6 +118,14 @@ export async function SubscribeAllEvents(
         } catch (error) {
             console.log("SubcommandError", error);
         }
+    } else {
```

Comment:
Thanks for fixing this case @Nabhag8848 . We must have missed this during implementation. 💯